### DISCUSSION TITLE
could not save html from plot_lignetwork version 2

### DISCUSSION THREAD

**Discussion by tjrong123** [chemosim-lab/ProLIF#170] [Q&A] [answered]
net=fp.plot_lignetwork(ligand_mol, kind="aggregate", threshold=0.2,
                          rotation=270)
net.save(os.path.join( output_name +".html"))
AttributeError: 'NoneType' object has no attribute 'Save'

**Comment by cbouy** [comment, marked as answer]
If you want to save the plot you have to use the `LigNetwork` class directly, like in your other issue #169:
```python
net = LigNetwork.from_fingerprint(fp, lig_mol,
threshold=float(threshold),
rotation=270)
net.save(os.path.join(output_name + ".html"))
```

`fp.plot_lignetwork` is only useful to display the plot in an interactive Jupyter Notebook, it doesn't return the underlying `LigNetwork` object

**Reply by tjrong123** [reply]
thanks it solved my probelm
